tools/appliance/systemvmtemplate/template.json

```diff
@@ -6,7 +6,14 @@
       "type": "shell",
       "execute_command": "echo 'cloud' | sudo -u root -S bash {{.Path}}",
       "scripts": [
-        "scripts/apt_upgrade.sh",
+        "scripts/apt_upgrade.sh"
+      ],
+      "expect_disconnect": true
```

Comment:
@weizhouapache this handles for reboot case. You can read more about it here https://www.packer.io/docs/provisioners/shell